CARDS-2056: PREMs Survey updates

marta- opened the pull request:
**Specs:**
* [CARDS-2056](https://phenotips.atlassian.net/browse/CARDS-2056) - _PREMs Survey updates_

**Testing:**
* Test all prems surveys as patient

**Reviewing:**
* commit by commit is recommended

[CARDS-2056]: https://phenotips.atlassian.net/browse/CARDS-2056?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

marta- committed:
CARDS-2056: PREMs Survey updates
* Start page for ED, IP, EDIP, Rehab: add “Please” before “Do not include other hospital stays…”

marta- committed:
CARDS-2056: PREMs Survey updates
* All questions in all prems surveys should be numbered
* CPESIC: descriptive text explaining why, for example, 12 comes after 10

marta- committed:
CARDS-2056: PREMs Survey updates
* CPESIC: Add min/max values to Year of Birth

marta- committed:
CARDS-2056: PREMs Survey updates
* Shorter module titles for ED, IP, Rehab

marta- committed:
CARDS-2056: PREMs Survey updates
* Do not display “This question is mandatory” under each question

marta- committed:
[misc] Code style: replaced indentation spaces with tabs

sashaandjic commented:
CPESIC - all changes are done as per request
------
![image](https://user-images.githubusercontent.com/23243244/218006878-28f9b2c0-9f3e-4101-8186-869f5c8d454d.png)
![image](https://user-images.githubusercontent.com/23243244/218006950-76807b7c-46df-4977-adaf-ae021fa59839.png)
![image](https://user-images.githubusercontent.com/23243244/218006979-2cc83bd9-e928-4b20-8c81-35510ec1a2bc.png)

This change just does not make sense:
![image](https://user-images.githubusercontent.com/23243244/218007065-bd50964e-aa2b-4719-9afd-48e0e113c6a6.png)

Review page does not look good with question numbers being added:
![image](https://user-images.githubusercontent.com/23243244/218007126-c4001de1-d1f6-44c9-be0e-f3d36dcd21ea.png)

sashaandjic commented:
Rehab survey - all changes done
![image](https://user-images.githubusercontent.com/23243244/218009350-f9413305-b191-4617-94f6-f8326ea97bd9.png)
![image](https://user-images.githubusercontent.com/23243244/218009323-fba5a1a5-50ed-45b5-8a88-22e0b508f7bb.png)

Question: In [JIRA](https://phenotips.atlassian.net/browse/CARDS-2056) task it said: Module 3 in Rehab should be labeled “Your Rehab Experience”. In the survey it is "Your rehabilitation experience"

sashaandjic commented:
ED & IP
-----
![image](https://user-images.githubusercontent.com/23243244/218010866-a4d59ca5-c952-48d0-8ae8-c5c5eeff7882.png)
![image](https://user-images.githubusercontent.com/23243244/218010888-6f981f99-2b75-4adc-8c69-783bb6cf8a57.png)

sashaandjic reviewed (approved):
tested

marta- commented:
> This change just does not make sense. It will just confuse the patient.

It is based on the paper template. Isn't it more confusing to see 12 come after 10?

> The Review page does not look good with question numbers being added:

One of the reasons numbering was requested was to make the review page more readable. Can you be more specific re what does not look good and do you have any suggestions of improvement?

> In [JIRA](https://phenotips.atlassian.net/browse/CARDS-2056) task it said: Module 3 in Rehab should be labeled **“Your Rehab Experience”.** In the survey it is **"Your rehabilitation experience"**

I was wrong on jira, implementation is right.

sashaandjic commented:
> One of the reasons numbering was requested was to make the review page more readable. Can you be more specific re what does not look good and do you have any suggestions of improvement?

This looks like some questions were skipped. We know why (the conditional questions), but the patient may think they did something wrong.

![image](https://user-images.githubusercontent.com/23243244/218143463-bc2d6d64-d43a-44ad-a15a-235a92ddafc3.png)

marta- commented:
> > One of the reasons numbering was requested was to make the review page more readable. Can you be more specific re what does not look good and do you have any suggestions of improvement?
> 
> This looks like some questions were skipped. We know why (the conditional questions), but the patient may think they did something wrong. However, troubleshooting will be much easier with numbers.
> 

And that's why there were indications while they were filling out the form that the next question would be n+m, not n+1

marta- committed:
CARDS-2056: PREMs Survey updates
* CPESIC: formatted some questions (in the template, some words are underlined - made them italic)

marta- committed:
CARDS-2056: PREMs Survey updates
* All surveys: merged the description of the "Overall..." question into the question text: “Overall… (please answer on a scale […])”

marta- committed:
CARDS-2056: PREMs Survey updates
Reverted the question numbering change

marta- commented:
Question numbers have been removed in the last commit.

marta- committed:
CARDS-2056: PREMs Survey updates
Updated the email templates for CPESIC to inform the recipient that they will be asked UP TO 49 questions instead of exactly 49

marta- commented:
The email templates for CPESIC have been updated in 88ba79e.

marta- committed:
CARDS-2056: PREMs Survey updates
Updated the pagination variants: progress for the longer surveys, text for the shorter ones

marta- commented:
Pagination variants updated per latest request. This PR is ready to be re-tested @sashaandjic

sdumitriu commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/CPESIC.xml:
```diff
@@ -5756,6 +5756,16 @@ Your response to this survey is voluntary but will provide us with important inf
+			</property>
+			<property>
+				<name>maxValue</name>
+				<value>2023</value>
```
Oh wow, precocious babies, filling in such long surveys...

sashaandjic reviewed (approved):
tested - all good

email template:
![image](https://user-images.githubusercontent.com/23243244/219143234-7ecf500b-13af-4ddd-99c4-de79bb1e7f22.png)
